layout/new.go

```diff
@@ -57,6 +57,15 @@ func NewImage(path string, ops ...ImageOption) (*Image, error) {
 		ri.createdAt = imageOpts.createdAt
 	}
 
+	if imageOpts.mediaTypes != "" {
+		ri.requestedMediaTypes = imageOpts.mediaTypes
+	} else {
+		ri.requestedMediaTypes = imgutil.OCITypes // without media types option, default to oci media type
```

Comment:
Actually, thinking about how this would work in the lifecycle, I do think we want an option that is like "I don't care, just use whatever media types are already there" ...so might update `imgutil.DefaultTypes` to be `default` instead of `""`.

Reply:
Updated in eaefdf2